pyproject.toml

```diff
@@ -1,3 +1,9 @@
 [build-system]
-requires = ["setuptools>=30.3.0", "wheel", "setuptools_scm[toml]>=3.4"]
+requires = ["setuptools>=64", "wheel", "setuptools_scm>=8.1"]
 build-backend = "setuptools.build_meta"
+
+# This has to be here for Mac builds to work.
+# It's a bit weird to have this here and all the other metadata in setup.py,
+# The right thing long term is to move it all to this file.
```

Comment:
Do you mean basically everything in `setup_args` in setup.py should go here?

Reply:
Basically.  Building packages from setup.py is being deprecated. The only thing we still have in our setup.py in RASG repos is the branch scheme code and cython plugin stuff for pyuvdata. Maybe look at pyradiosky as a guide since it doesn't have any cython.